### PR TITLE
Update react example

### PR DESCRIPTION
**Pull request by ambarwootric**
Preview:
react-example-survey

For testing refer https://github.com/Wootric/spa-examples/blob/4dca2e92fbaf63784084fa0eaea21843ae133569/react-example/README.md

Trello: https://trello.com/c/Wwhp66lf/4821-check-for-outdated-docs-and-examples

### CODE REVIEW

File: react-example/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+```
+git clone https://github.com/Wootric/spa-examples.git
+cd spa-examples
+cd react-example
+yarn install/npm install
+````
+
+### Setup Wootric (using Wootric component)
+1. Check `src/app/Wootric.js` for main setup and update `WOOTRIC_ACCOUNT_TOKEN` at `src/config/defaults.js` with your `account_token`.
```
Comment (hachibu) [inline react-example/README.md]:
```suggestion
1. Check `src/Wootric.js` for main setup and update `WOOTRIC_ACCOUNT_TOKEN` at `src/config/defaults.js` with your `account_token`.
```

File: react-example/src/config/defaults.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const WOOTRIC_ACCOUNT_TOKEN = 'NPS-3bbb4510'; // This is your unique account token.
```
Comment (hachibu) [inline react-example/src/config/defaults.js]: Change to fake token...
```suggestion
export const WOOTRIC_ACCOUNT_TOKEN = 'NPS-XXXXXX'; // This is your unique account token.
```

File: react-example/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+2. Using the component: Add component `Wootric` to `src/app/App.js` (here like at line no. 23).
```
Comment (hachibu) [inline react-example/README.md]:
```suggestion
2. `Wootric` component has been added to `src/App.js` on line 23.
```

**Comment by hachibu** [comment]
Screen Shot 2019-09-23 at 2 31 33 PM

I got it working locally!

**Comment by ambarwootric** [comment]
Thank you so much @hachibu, I really missed such mistakes but it feels good to know that you looked in detail and suggested changes 🙏

**Pull request merged**